content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -22,38 +22,43 @@ At the end of the tutorial you'll have a new O3DE project based on the default p
 
-This tutorial uses the following project name and directories in the examples:
+This tutorial uses the following project name and directories in the examples. (Depending on how you set up O3DE, you might not have all of these directories.)
 
-* O3DE engine install directory: `C:\o3de-install`
+* O3DE engine source: `C:\o3de`
+* Installed O3DE engine (contains pre-built SDK engine binaries): `C:\o3de-install`
 * New project name and location: `C:\o3de-projects\MyProject`
-* Package directory (created earlier during [setup](/docs/welcome-guide/setup/setup-from-github/#build-the-engine)): `C:\o3de-packages`
+* Package directory (created during [set up from GitHub](/docs/welcome-guide/setup/setup-from-github/#build-the-engine)): `C:\o3de-packages`
 
-1. Open a command line window and change to your O3DE engine install directory. The `o3de-install` directory created in [Setting up O3DE from GitHub](/docs/welcome-guide/setup/setup-from-github) is shown in this example.
+1. Open a command line window and change to your O3DE engine directory.
 
-    ```cmd
-    cd C:\o3de-install
-    ```
+    1. If you set up your engine as a [source engine](/docs/welcome-guide/setup/setup-from-github/#build-the-engine), use the engine source directory.
 
-1. Use the `o3de` script in the `scripts` subdirectory to create a new project. The `create-project` command, used with the `project-path` and no other options, creates a new project using the default project template.
+        ```cmd
+        cd C:\o3de
+        ```
 
-    ```cmd
-    scripts\o3de.bat create-project --project-path C:\o3de-projects\MyProject
-    ```
+    1. Or, if you installed O3DE or built your engine as a [SDK engine](/docs/welcome-guide/setup/setup-from-github/#build-the-engine) using the `INSTALL` target, use the installed engine directory.
+
+        ```cmd
+        cd C:\o3de-install
+        ```
 
-1. Register the engine to the project. This adds your new project to the list of known projects in the O3DE manifest, located at `<USER_DIRECTORY>/.o3de/o3de_manifest.json`. It also enables O3DE **Project Manager** to be aware of your project. Note that the full (absolute) path to the project is always needed for this command.
+1. Use the `o3de` script in the `scripts` subdirectory to create a new external project. The `create-project` command, used with the `project-path` and no other options, creates a new project using the default project template. This command also registers the engine to the project in the project's `project.json` manifest.
 
     ```cmd
-    scripts\o3de.bat register --project-path C:\o3de-projects\MyProject
+    scripts\o3de.bat create-project --project-path C:\o3de-projects\MyProject
     ```
 
+    Additionally, this command registers the project, adding it to the list of known projects in the O3DE manifest located in `<USER_DIRECTORY>/.o3de/o3de_manifest.json`, and making **Project Manager** aware of your project.
```

Comment:
There is a new feature that I just [submitted](https://github.com/o3de/o3de/pull/2967) today, that I believe it isn't big enough to document in the welcome-guide and will only confuse users, but it is relevant technical information.

If the path for a created project happens to be within the engine directory, the project is instead registered to that engine's engine.json file instead of the o3de_manifest.json file.

That being said **Project Manager** will still be aware of that project. The reason for registering a project that is created in the engine directory with the engine.json file is an "ease of use" optimization.

Reply:
Thanks! Sounds like good info for the User Guide section on project configuration. @chanmosq FYI.